cli/src/modules/workflow.rs

```diff
@@ -20,7 +26,13 @@ pub struct WorkflowOptions {
 pub enum WorkflowSubcommands {
     CredentialOffer {
         #[clap(long, short)]
-        connection_id: String,
+        connection_id: Option<String>,
+
+        #[clap(long, short)]
+        timeout: Option<u32>,
+
+        #[clap(long = "self", short = 's')]
+        sent_to_self: bool,
```

Comment:
```suggestion
        send_to_self: bool,
```

WDYT about shortening to `self` and adding docs about the intended use via `clap(...about=...)`?

Reply:
self is a reserved keyword and does not enjoy me using it.